ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -259,8 +262,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-upi
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|upi/.*|pkg/.*|images)/(alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

Comment:
Shouldn't we run this for some changes in the pkg files too? Since we run create ignition-configs in most UPI tests.

Reply:
> Shouldn't we run this for some changes in the pkg files too? Since we run create ignition-configs in most UPI tests.

I think most errors coming out of those changes that would break upi would break ipi too. 

In general, I would like to be scaling back the number of seemingly unrelated tests we're running on changes until we're getting a good signal, then we can dial up the number of tests we're running.